src/pages/AllMoviesPage.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <div>
+    <h1>All Movies Page</h1>
+    <router-link to="/movie/1">Movie Number 1</router-link>
+    <a href="" @click="movie2">Movie Number 2</a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "AllMoviesPage",
+  methods: {
+    movie2() {
```

Comment:
Also, a more explicit method name might be better here, something like `goToMovie2` for example - this makes it clearer for others (and us, in the future) to see what the code does at a glance 🚀